packages/web-app-files/src/components/Sidebar.vue

```diff
@@ -11,25 +13,28 @@
       </div>
       <div class="uk-inline">
         <div class="uk-flex uk-flex-middle">
-          <span
+          <h3
```

Comment:
@kulmann this should be `<h2>` according to the accessibility call, right?

Reply:
Makes sense IMO, I'll go for it for now. We can still revert it later if `h3` was correct.